### DISCUSSION TITLE
Question: Can you display videos?

### DISCUSSION THREAD

**Discussion by VeryUsual** [InfiniTimeOrg/InfiniTime#1809] [Q&A] [answered]
Can you display a video from a video file (example: .mp4 file) on the PineTime?

**Comment by JF002** [comment, marked as answer]
Short answer : no.

Less short answer : In InfiniTime : no

Long answer : It all depends on how much programming effort you're willing to put in ;-) The MCU that powers the PineTime is running at 64Mhz and has 64KB of RAM and 512KB of flash memory. Then, the SPI bus to drive the display is limited to 8Mhz, which limits the framerate to ~10FPS (using 16-bits colors).
So storing a video file, decoding it and displaying it in real time won't be easy on the PineTime. Not easy, but maybe not impossible! For example, @TT-392 managed to play bad apples on the PineTime using various hacks and tricks : https://www.youtube.com/watch?v=RqL8V3xWmxg
